### PR TITLE
Admin View List Menu (Update Search)

### PR DESCRIPTION
**Pull request by hungbkdn**
Update search into this function

### CODE REVIEW

File: app/Http/Controllers/DailyMenuController.php [hunk 1]
```diff
@@ -29,12 +29,21 @@ public function __construct(DailyMenu $dailyMenu)
...
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
-        $listDailyMenu = $this->dailyMenu->select('date')->distinct()->orderBy('date', 'desc')->paginate(10);
-        
-        return view('daily_menus.index', ['listDailyMenu' => $listDailyMenu]);
+        if ($request->has('date')) {
+            $listDailyMenu = $this->dailyMenu->select('date')
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: Variavle name `$listDailyMenu` should by `$dailyMenus`

Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: You can assign variable before check `date`
Example:
```
$dailyMenus = $this->dailyMenu->select('date');
if ($request->has('date')) {
    $dailyMenus = $dailyMenus->where('date', $request['date']);
}
$dailyMenus = $dailyMenus->distinct()->orderBy('date', 'desc')->paginate(10);
```

File: resources/views/daily_menus/index.blade.php [hunk 2]
```diff
@@ -20,6 +20,18 @@
             <div class="dataTables_wrapper form-inline dt-bootstrap">
                 <div class="row">
                     <div class="col-sm-12">
+                        <div class="form-group">
+                            <form method="GET" action="{{ route('daily-menus.index') }}">
+                                <label for="dateSearch">{{ _('Choose Date') }}: </label>
+                                @if(!empty($date))
```
Comment (ghost) [inline resources/views/daily_menus/index.blade.php]: You can check `$date` like this:
```
{{ (!empty($date)) ? $date : "" }}
```

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by hungbkdn** [comment]
I have updated, please check it again
Thank you so much!

File: resources/views/daily_menus/index.blade.php [hunk 3]
```diff
@@ -20,6 +20,14 @@
             <div class="dataTables_wrapper form-inline dt-bootstrap">
                 <div class="row">
                     <div class="col-sm-12">
+                        <div class="form-group">
+                            <form method="GET" action="{{ route('daily-menus.index') }}">
+                                <label for="dateSearch">{{ _('Choose Date') }}: </label>
+                                <input type="date" class="form-control" id = "dateSearch" name="date"
+                                    value="{{ (!empty($date)) ? $date : "" }}" style="width: 20em">
+                                <input type="submit" class="btn-primary btn" value="Search">
```
Comment (ghost) [inline resources/views/daily_menus/index.blade.php]: Please call *Search* by translation strings

File: resources/views/daily_menus/index.blade.php [hunk 4]
```diff
@@ -20,6 +20,14 @@
             <div class="dataTables_wrapper form-inline dt-bootstrap">
                 <div class="row">
                     <div class="col-sm-12">
+                        <div class="form-group">
+                            <form method="GET" action="{{ route('daily-menus.index') }}">
+                                <label for="dateSearch">{{ _('Choose Date') }}: </label>
```
Comment (ghost) [inline resources/views/daily_menus/index.blade.php]: Use double underscore `__` to unified with other files.

**Review by ghost — COMMENTED** [review]
Please check my comments about translation strings

**Comment by hungbkdn** [comment]
I have updated, please check it again
Thank you so much!

**Comment by ghost** [comment]
LGTM :innocent:

**Pull request merged**